src/Compression/Compression.cs

```diff
@@ -0,0 +1,153 @@
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+
+using ObjCRuntime;
+
+namespace Compression {
+
+
+	public partial class CompressionStream : IDisposable {
```

Comment:
This is missing availability attributes
It seems new in iOS 9, tvOS 9 and macOS 10.11 (and part of watchOS 2.0 so its not needed)